commons/all/src/main/java/org/infinispan/commons/util/ProgressTracker.java

```diff
@@ -0,0 +1,171 @@
+package org.infinispan.commons.util;
+
+import java.time.Instant;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.locks.Lock;
+import java.util.concurrent.locks.ReentrantLock;
+
+import org.infinispan.commons.logging.Log;
+import org.infinispan.commons.logging.LogFactory;
+import org.infinispan.commons.time.TimeService;
+
+import net.jcip.annotations.GuardedBy;
+import net.jcip.annotations.ThreadSafe;
+
+@ThreadSafe
+public final class ProgressTracker {
+
+   private static final Log log = LogFactory.getLog("LIFECYCLE");
```

Comment:
Technically this should be in https://github.com/infinispan/infinispan/blob/main/commons/all/src/main/java/org/infinispan/commons/logging/Log.java#L57-L59 instead.